catalog/opensearch/catalog-opensearch-source/src/main/java/ddf/catalog/source/opensearch/OpenSearchSource.java

```diff
@@ -629,6 +630,8 @@ private InputTransformer getInputTransformer(InputStream inputStream) {
 
     protected InputTransformer lookupTransformerReference(String namespaceUri)
             throws InvalidSyntaxException {
+        LOGGER.debug("Looking up Input Transformer by schema : {}", namespaceUri);
```

Comment:
Maybe this should be trace as it's going to be logged in the valid handling as well.